### DISCUSSION TITLE
Where would you like to trip in Japan?

### DISCUSSION THREAD

**Discussion by r-nishioka** [NAIST-SE/AttractingContributors#1159] [2024 Attraction Contributions]
https://github.com/r-nishioka/AttractingContributors-Trip

**Comment by ryosuke-sota** [comment]
I made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Comment by kohanaist** [comment]
I made a PR on your repo. 
Could you contribute to mine? 
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1148
